services/bookables-api/src/helpers/searchAdministrators.ts

```diff
@@ -1,16 +1,24 @@
 import to from 'await-to-js';
 
-import * as request from '../../../libs/request';
-import params from '../../../libs/params';
-import config from '../../../config';
+import * as request from '../libs/request';
+import params from '../libs/params';
+import config from '../libs/config';
 
 const URI_SOURCE = 'misc/getGroupMembers';
 
-async function searchAdministrators(body) {
+export interface SearchAdministratorsResult {
+  data?: Record<string, unknown>;
+}
+
+async function searchAdministrators(body: {
+  email?: string;
+  mailbox?: string;
+}): Promise<SearchAdministratorsResult | undefined> {
   const [readError, { datatorgetEndpoint, apiKey }] = await to(
     params.read(config.datatorget.envsKeyName)
   );
   if (readError) {
+    // eslint-disable-next-line no-console
```

Comment:
perhaps we should take the opportunity to add the log function instead of console.error here?

Reply:
Well... console log is used everywhere. I'd rather do that in a refactoring task while introducing a log wrapper to the lambas to avoid passing around the was context too ...